### PR TITLE
Use tagged images when deploying Dispatch

### PR DESCRIPTION
**Pull request by jbarrick-mesosphere**

### CODE REVIEW

**Comment by jbarrick-mesosphere** [comment]
Still testing this locally, but this should resolve the issues with the airgapped deployments.

**Comment by jbarrick-mesosphere** [comment]
Works locally!

File: addons/dispatch/1.2.x/dispatch-4.yaml [hunk 1]
```diff
@@ -53,6 +53,16 @@ spec:
     version: "1.2.0"
     values: |
       ---
+      dispatch:
+        image: mesosphere/dispatch:1.2.0
+        testImage: mesosphere/dispatch-helm-test:1.2.0
+      tekton-dashboard:
+        image:
+          dashboard: gcr.io/tekton-releases/github.com/tektoncd/dashboard/cmd/dashboard:v0.7.0
```
Comment (chhsia0) [inline addons/dispatch/1.2.x/dispatch-4.yaml]: s/`v0.7.0`/`v0.6.1.5`/

File: addons/dispatch/1.2.x/dispatch-4.yaml [hunk 2]
```diff
@@ -53,6 +53,16 @@ spec:
...
+      tekton:
+        controller:
```
Comment (chhsia0) [inline addons/dispatch/1.2.x/dispatch-4.yaml:63]: @gpaul Could you tag `docker.io/gpaul/controller-10a3e32792f33651396d02b6855a6e36@sha256:61ce6412470881e83180399a8ea116258865da20e612914734e6b8c09b5a31fd` and add it here?

**Review by chhsia0 — CHANGES_REQUESTED** [review]

File: addons/dispatch/1.2.x/dispatch-4.yaml (hunk 2 above)
Comment (jbarrick-mesosphere) [reply addons/dispatch/1.2.x/dispatch-4.yaml:63]: doh good catch

Comment (jbarrick-mesosphere) [reply addons/dispatch/1.2.x/dispatch-4.yaml:63]: fixed, i'm going to re-run the test now though

Comment (jbarrick-mesosphere) [reply addons/dispatch/1.2.x/dispatch-4.yaml:63]: tests are happy still

**Comment by dkoshkin** [comment]
Collected the images with this branch.
The only images that could not be collected were the test ones:
```
These images will not work for air-gapped installations:
  docker.io/mesosphere/dispatch-cue@sha256:0a0f06150078f9413c67d60d3b3e3959de6d6e7c1c6455e8f79aa374a3db095f
  docker.io/mesosphere/dispatch-starlark@sha256:10c1d234c2bd44846ff94f05b35ab54bb3ac8bd22b4196a8cebfce1be117b259
```

```
[
  {
    "scheme": "https",
    "registry": "docker.io",
    "image": "alpine/git",
    "tag": "latest"
  },
  {
    "scheme": "https",
    "registry": "docker.io",
    "image": "argoproj/argocd",
    "tag": "v1.5.7"
  },
  {
    "scheme": "https",
    "registry": "docker.io",
    "image": "cederstone/git-daemon",
    "tag": "latest"
  },
  {
    "scheme": "https",
    "registry": "docker.io",
    "image": "google/cloud-sdk",
    "tag": "255.0.0-alpine"
  },
  {
    "scheme": "https",
    "registry": "docker.io",
    "image": "jbarrickmesosphere/webhook-d4749e605405422fd87700164e31b2d1",
    "tag": "latest"
  },
  {
    "scheme": "https",
    "registry": "docker.io",
    "image": "lachlanevenson/k8s-kubectl",
    "tag": "latest"
  },
  {
    "scheme": "https",
    "registry": "docker.io",
    "image": "library/alpine",
    "tag": "latest"
  },
  {
    "scheme": "https",
    "registry": "docker.io",
    "image": "library/golang",
    "tag": "1.13.0-buster"
  },
  {
    "scheme": "https",
    "registry": "docker.io",
    "image": "library/redis",
    "tag": "5.0.3"
  },
  {
    "scheme": "https",
    "registry": "docker.io",
    "image": "mesosphere/dispatch-cue",
    "tag": "v0.3"
  },
  {
    "scheme": "https",
    "registry": "docker.io",
    "image": "mesosphere/dispatch-entrypoint",
    "tag": "1.2.0"
  },
  {
    "scheme": "https",
    "registry": "docker.io",
    "image": "mesosphere/dispatch-helm-test",
    "tag": "1.2.0"
  },
  {
    "scheme": "https",
    "registry": "docker.io",
    "image": "mesosphere/dispatch-starlark",
    "tag": "v0.5-beta2"
  },
  {
    "scheme": "https",
    "registry": "docker.io",
    "image": "mesosphere/dispatch-tekton-controller",
    "tag": "1.2.0"
  },
  {
    "scheme": "https",
    "registry": "docker.io",
    "image": "mesosphere/dispatch",
    "tag": "1.2.0"
  },
  {
    "scheme": "https",
    "registry": "docker.io",
    "image": "mesosphere/kommander",
    "tag": "3.104.4"
  },
  {
    "scheme": "https",
    "registry": "docker.io",
    "image": "minio/mc",
    "tag": "RELEASE.2020-03-06T23-29-45Z"
  },
  {
    "scheme": "https",
    "registry": "docker.io",
    "image": "minio/minio",
    "tag": "RELEASE.2020-03-06T22-23-56Z"
  },
  {
    "scheme": "https",
    "registry": "gcr.io",
    "image": "tekton-releases/github.com/tektoncd/dashboard/cmd/dashboard",
    "tag": "v0.6.1.5"
  },
  {
    "scheme": "https",
    "registry": "gcr.io",
    "image": "tekton-releases/github.com/tektoncd/pipeline/cmd/creds-init",
    "tag": "v0.11.1"
  },
  {
    "scheme": "https",
    "registry": "gcr.io",
    "image": "tekton-releases/github.com/tektoncd/pipeline/cmd/git-init",
    "tag": "v0.11.1"
  }
]
```

**Review by dkoshkin — APPROVED** [review]

**Review by chhsia0 — APPROVED** [review]

**Pull request merged**